orthanc/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Linux for Health Orthanc DICOM Server image.
+#
+# Orthanc is a DICOM server, developed and maintained by Sébastien Jodogne. Full 
+# documentation is available in the Orthanc Book: https://book.orthanc-server.com
+#
+# Orthanc is installed in /usr/sbin.
+#
+
+FROM ubuntu:18.04
+
+LABEL maintainer="Linux for Health"
+LABEL com.linuxforhealth.component="orthanc image"
+LABEL name="orthanc-image"
+LABEL com.linuxforhealth.license_terms="https://www.apache.org/licenses/LICENSE-2.0"
+LABEL summary="Linux For Health Orthanc DICOM Image"
+LABEL description="Orthanc, free cross-platform DICOM server"
+LABEL vendor="The Orthanc project"
+LABEL maintainer2="Sebastien Jodogne"
+
+RUN apt -y clean && \
+    apt -y update && \
+    apt -y upgrade && \
+    apt -y install orthanc
+
+COPY ./orthanc.json /etc/orthanc
+
+VOLUME [ "/var/lib/orthanc/db-v6" ]
+EXPOSE 4242
+EXPOSE 8042
+
+ENV TZ=America/New_York
+
+ENTRYPOINT [ "Orthanc" ]
+CMD [ "/etc/orthanc/" ]
```

Comment:
I was able to run the container in the LFH Connect compose stack, and validate that it works 🎉 !
I was curious about the CMD here. Do we need the trailing slash?

Reply:
I believe we do, because the config filename is likely appended on Orthanc startup.

Reply:
I'll check this out and if it can be removed, I'll do it as a part of the next connect commit for an Orthanc route.